fastlane/Fastfile

```
@@ -306,36 +305,62 @@ platform :ios do
+      UI.message("Workspace exists at #{workspace_path}")
 
+      # Verify export_options.plist exists
+      export_options_path = File.join(project_root, "ios", "export_options.plist")
+      unless File.exist?(export_options_path)
+        UI.user_error!("Export options plist does not exist at #{export_options_path}")
+      end
+      UI.message("Export options plist exists at #{export_options_path}")
+
+      # Call build_app and capture the result
+      begin
+        result = build_app(
```

rubocop

[warning] Useless assignment to variable - `result`.

(Lint/UselessAssignment)